docs/user-guide/custom-themes.md

```diff
@@ -677,6 +677,15 @@ Bootswatch theme].
 [MkDocs Bootstrap theme]: https://mkdocs.github.io/mkdocs-bootstrap/
 [MkDocs Bootswatch theme]: https://mkdocs.github.io/mkdocs-bootswatch/
 
+!!! Warning
```

Comment:
I see a problem in this warning because it is only helpful to a fraction of the users. Any user who starts with a new template does not need this warning.

I would suggest to create a separate migration guide for this type of information.

Reply:
Thats a good point, I will be solving these issues in the 2 child themes. 

I'll remove this warning from, custom-themes.md, and remove the paragraph about the bootstrap themes from styling-your-docs.md.

Or do you think it doesn't make sense to mention the filename changes in these docs at all?

Reply:
Yeah, lets remove this altogether. This sort of information goes in the release notes, not here.

In addition this is information which should be directed at authors of themes which inherit from the `mkdocs` theme, not users of those themes. We don't expect the users to modify the theme. Instead we expect the theme author to update the theme and the users to update to the new version.

Reply:
Ah yeah ok that makes sense, I'll take this out.

Reply:
Also is updating the release notes something I can do? Or should I just close this PR and make the updates to the themes that inherit?

Reply:
Add a "Backward Incompatible Changes" sub-section to the 1.1 section of [docs/about/release-notes.md][1]. See the [similar section][2] for version 1.0 for an example. You can either update this PR or close it and open a new one, whichever is easier.

[1]: https://github.com/mkdocs/mkdocs/blob/master/docs/about/release-notes.md
[2]: https://github.com/mkdocs/mkdocs/blob/master/docs/about/release-notes.md#backward-incompatible-changes

Reply:
I'll just open a new one.